src/file-events/cpp/apple_fsnotifier.cpp

```diff
@@ -145,7 +145,7 @@ void Server::handleEvents(
 }
 
 void Server::handleEvent(JNIEnv* env, char* path, FSEventStreamEventFlags flags) {
-    logToJava(FINE, "Event flags: 0x%x for %s", flags, path);
+    logToJava(FINER, "Event flags: 0x%x for '%s'", flags, path);
```

Comment:
Why are we using `FINER` here? That would be `TRACE` on the Gradle side, and we can't set this log level.

Reply:
That's a good point. I'll revert and instead bump the test log level to `INFO`.